ingestion/src/metadata/ingestion/source/dashboard/superset/connection.py

```diff
@@ -60,7 +60,7 @@ def test_connection(client: Union[SupersetAPIClient, Engine]) -> None:
     """
     try:
         if isinstance(client, SupersetAPIClient):
-            client.fetch_menu()
+            client.fetch_total_dashboards()
```

Comment:
did this change because `fetch_menu` requires additional permissions, also we are not using fetch menu api anywhere else